editor/modes/visual-editor/block.js

```diff
@@ -364,6 +376,19 @@ class VisualEditorBlock extends Component {
 						<div className="editor-visual-editor__block-controls">
 							<BlockSwitcher uid={ block.uid } />
 							<Slot name="Formatting.Toolbar" />
+							<Toolbar className="editor-visual-editor__mobile-tools">
+								{ ( showUI || isHovered ) && <BlockMover uids={ [ block.uid ] } /> }
```

Comment:
Curious if we need both BlockMover here and above, or if we could apply styles to the root element when `'is-showing-mobile-controls'` is applied to appear as it does with the `Toolbar` component.